src/main/java/gregtech/common/metatileentities/MetaTileEntityClipboard.java

```diff
@@ -195,25 +208,44 @@ public boolean onRightClick(EntityPlayer playerIn, EnumHand hand, EnumFacing fac
                 MetaTileEntityUIFactory.INSTANCE.openUI(getHolder(), (EntityPlayerMP) playerIn);
             }
         } else {
-            if(!getWorld().isRemote) {
-                BlockPos pos = this.getPos(); // Saving this for later so it doesn't get mangled
-                World world = this.getWorld(); // Same here
+            breakClipboard(playerIn);
+        }
+        return true;
+    }
+
+    private void breakClipboard(@Nullable EntityPlayer player) {
+        if (!getWorld().isRemote) {
+            BlockPos pos = this.getPos(); // Saving this for later so it doesn't get mangled
+            World world = this.getWorld(); // Same here
 
-                NonNullList<ItemStack> drops = NonNullList.create();
-                getDrops(drops, playerIn);
+            NonNullList<ItemStack> drops = NonNullList.create();
+            getDrops(drops, player);
 
-                Block.spawnAsEntity(playerIn.world, this.getPos(), drops.get(0));
-                this.dropAllCovers();
-                this.onRemoval();
+            Block.spawnAsEntity(getWorld(), this.getPos(), drops.get(0));
+            this.dropAllCovers();
+            this.onRemoval();
 
-                world.removeTileEntity(pos);
-                world.setBlockState(pos, Blocks.AIR.getDefaultState(), 3);
-                this.scheduleRenderUpdate();
+            world.setBlockState(pos, Blocks.AIR.getDefaultState(), 3);
+            this.scheduleRenderUpdate();
```

Comment:
setBlockState with flag 3 should sync to client and ask for a render update, would `scheduleRenderUpdate` be needed?